FSCalendar/FSCalendar.m

```diff
@@ -995,7 +995,7 @@ - (CGFloat)preferredRowHeight
         CGFloat contentHeight = self.transitionCoordinator.cachedMonthSize.height-headerHeight-weekdayHeight-_scopeHandle.fs_height;
         CGFloat padding = 5;
         if (!self.floatingMode) {
-            _preferredRowHeight = (contentHeight-padding*2)/6.0;
+            _preferredRowHeight = (contentHeight-padding*2);
```

Comment:
This works for calendars that are set to `FSCalendarScopeWeek`, however, once you set it to `FSCalendarScopeMonth`, you end up having 1 row for the Month Scope.

Perhaps an additional scope check would help here?
```
            _preferredRowHeight = (contentHeight-padding*2);
            if (_scope == FSCalendarScopeMonth) {
                _preferredRowHeight /= 6.0;
            }
```